### ISSUE TITLE
Portable Bunkhouse Key is a myth

### ISSUE THREAD

**Issue by refringe** [sp-tarkov/server#610]
## **General information**:

**Project issue is for:** </br>
*(Server/Modules/Launcher)*
No Idea what to put here :D

**Expected result**:</br>
*(What you expect to happen)*
Find the Bunkhouse key on scavs pockets/backpacks.

**Actual result**:</br>
*(What actually happened)*
Never got them to spawn unless I ran Mods. I've been playing SPT for over a year now and I've done 7-10 Hardcore runs to lvl 55-60. I feel like some key drops are actually set to 0. On my last 7-8 runs I haven't found the Bunkhouse key for customs at all. 250+ raids on each hardcore. The Key spawn in factory is so low that I ran over 400-500 factory runs just to get it to spawn but haven't managed to get it unless I used a Mod to buff the spawn rate. The drop rate from scavs pockets/backpacks is almost 0.
I know this is not a priority as you guys are working on the new update but would you guys look into it if you get a chance? I don't wanna install mods to make the spawn rate higher as that kills the gameplay and fun of the grind. I'm not the only one that had this. There's multiple people that say that the key doesn't exist in the drop pool.The same goes with the Operating room Key on Lighthouse. The only way to get these two keys is to find a rusted key from Streets of Tarkov and do ALT+f4 Runs until you get these two to spawn on the floor.

*(Add a description of the issue here)*

## **How to reproduce**:
1. RUN SPT.
2. Play for 2-3 weeks 12-15 hours a day.
3. Give up because you can't get a key to spawn or find on scavs/PMC.

## **Log files**:
Server: *(found in `/user/logs/server-yyyy-mm-dd-xx.log`)*
BepinEx: *(found in `/BepinEx/LogOutput.log`)*
Client: *(found in `/Logs/log_<date>_<version>/<date>_<version> traces.log`)*

## **Profile**:
You can optionally include your game profile if you think it would help us reproduce the error easily *(found in `/user/profiles/xxxxxxxxxxxx.json`)*
The profile id can be found in the top left of the launcher.

## Screenshots:
*(Optional, only add them if you think it adds value)*

## Notes - PLEASE READ!
***DO NOT INCLUDE THIS SECTION IN THE REPORT***
- Do not include any of the text written in parentheses and italics in your report *(example text - do not include me in the report!)*
- If you uploaded and attach the log files to the issue report, instead of using the codepaste, please remove the "Log Files" section as well.
- If you use the codepaste site please make sure the paste has a sufficiently long lifespan set, 24hrs is often not enough.
- You must be able to replicate the issue with a fresh profile, while running *<ins>no</ins>* mods.
- If you are using a profile from an older version of SPT-AKI, then please make a fresh profile and replicate the issue like above prior.
- You must upload *<ins>all</ins>* the required log files, even if you think they are useless.
- Failure to comply with *ANY* of the requirements may result in your issue being closed without further investigation.

---

[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/345) by _**GeorgicaBaws**_

**Comment by refringe** [comment]
You provided no logs so I can't tell if you're talking about 3.8.0 or 3.7.6 but with the amount of playtime you're referencing I'm feeling it's 3.7.6 and after checking the loose loot table for Customs on 3.7.6 I can tell you there are exactly two spawns for the Portable Bunkhouse key on Customs alone. 

You also didn't read the notes section where it asks to please read...

We need more info.

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/345#issuecomment-22116) by _**Valens**_

**Comment by refringe** [comment]
My bad! It's my first time posting around here and didn't know what to include in the information. Yes, I'm talking about 3.7.6. I have only once managed to get the portable bunkhouse key and that was on customs on my first playthrough. Where are these spawns exactly? On scavs/jackets/drawers? I've looted plenty of those in my runs as customs is on of my most played maps but still haven't found it :( I keep finding oli/goshan/idea keys and the Power key.

I have attached the files that are requested.

Please let me know if you require anymore information.

---
### Attachments
- [server-2024-01-27.log](https://dev.sp-tarkov.com/attachments/d9b5816d-c4fb-4a51-8426-5675114fcfa5) (986 KiB)
- [LogOutput.log](https://dev.sp-tarkov.com/attachments/5c5716fd-396a-4dc4-be92-fb105c1bca67) (7 KiB)
- [2024.01.27_16-54-39_0.13.5.3.26535 traces.log](https://dev.sp-tarkov.com/attachments/58a7140e-fd92-4713-88a0-18474c1dcb5c) (45 KiB)


---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/345#issuecomment-22119) by _**GeorgicaBaws**_

**Comment by refringe** [comment]
Can't tell you exactly where as that would require tools I don't have setup atm.

I can show you the loose loot spawns of it on Customs however, and that it's pretty rare in the two spots it does show up.
```

{"_id":"-1581902931","_tpl":"5938144586f77473c2087145"}
{"key":"-1581902931"},"relativeProbability":1}
{"key":"-1581902931"},"relativeProbability":1}
```

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/345#issuecomment-22120) by _**Valens**_

**Comment by refringe** [comment]
I assume one of them is giving tree spot and the other one is near trailer park in the garbage pile. I guess I'm just unlucky? I wish there was an even % of getting it as it's not a big key like the marked ones. I find 5-10 keys per reserve run (doing bunker runs and getting all filing cabinets), most of them are the oli/goshan/idea and the Power key. The power key is the most common and I find 3-4 per raid. Thank for the information. I see that they added a barter for the bunkhouse key in 3.8.0 so it won't be a problem anymore. I have suffered enough! :D

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/345#issuecomment-22121) by _**GeorgicaBaws**_

**Comment by refringe** [comment]
All I can say is this is based off of just over a hundred thousand dumps of offline raids. So it's unfortunately as good as we're going to get loot wise.

Hardcore runs unfortunately suffer for it.

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/345#issuecomment-22132) by _**Valens**_

**Comment by refringe** [comment]
Ok! Thank you for the information! I appreciate it! You can mark the issue as resolved if you want :) or if I have to do it, please let me know how to! Have a great day!

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/345#issuecomment-22133) by _**GeorgicaBaws**_
